### PR TITLE
https://issues.redhat.com/browse/ACM-7770

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-7770

### CODE REVIEW

**Review by thuyn-581 — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_install.adoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ The cluster backup and restore operator is not installed automatically. Continue
 
 - The backup component works with the OADP Operator that is installed in the component namespace. 
 
+*Warning:*
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_install.adoc]: Based on the [Special tagging doc](https://github.com/stolostron/rhacm-docs/blob/gh-pages/_pages/wr_special_tag_guidance.md), i would change this to "Important"

File: business_continuity/backup_restore/backup_install.adoc [hunk 2]
```diff
@@ -11,6 +11,11 @@ The cluster backup and restore operator is not installed automatically. Continue
...
+
+ * If you try to manually install the OADP  Operator on the hub cluster, your OADP Operator and Velero custom resource definition (CRD) versions must exactly match the `MultiClusterHub` OADP Operator and Velero CRD versions. If these versions do not exactly match one another, you encounter problems.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_install.adoc]:
```suggestion
 * If you try to manually install the OADP Operator on the hub cluster, your OADP Operator and Velero custom resource definition versions must exactly match the `MultiClusterHub` OADP Operator and Velero custom resource definition versions. If these versions do not exactly match one another, you encounter problems. 
```

File: business_continuity/backup_restore/backup_install.adoc [hunk 3]
```diff
@@ -11,6 +11,11 @@ The cluster backup and restore operator is not installed automatically. Continue
 
 - The backup component works with the OADP Operator that is installed in the component namespace. 
 
+*Warning:*
+
+ * If you try to manually install the OADP  Operator on the hub cluster, your OADP Operator and Velero custom resource definition (CRD) versions must exactly match the `MultiClusterHub` OADP Operator and Velero CRD versions. If these versions do not exactly match one another, you encounter problems. 
+ * CRDs are cluster-scoped, so you cannot have two different versions of an OADP Operator or Velero CRD installed on the same cluster. If you have two different versions, one version runs with the wrong CRD. If you have previously installed and used the OADP Operator on the hub cluster in a namespace different from the backup component namespace, uninstall this version.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_install.adoc]: It seems like this is similar to the first bullet point. Is the second bullet point suppose to be extra info to explain the requirement? 

would it be helpful to combine the statements?

"*Important:* If you manually install the OADP operator, the custom resource definition version of the OADP operator and Velereo must exactly match. If these versions do not exactly match one another, you encounter problems. If you have previously installed and used the OADP Operator on the hub cluster in a namespace different from the backup component namespace, uninstall this version.

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_install.adoc [hunk 4]
```diff
@@ -11,6 +11,10 @@ The cluster backup and restore operator is not installed automatically. Continue
 
 - The backup component works with the OADP Operator that is installed in the component namespace. 
 
+*Important:*
+
+If you manually install the OADP operator, the custom resource definition version of the OADP operator and Velereo must exactly match. If these versions do not exactly match one another, you encounter problems. If you have previously installed and used the OADP Operator on the hub cluster in a namespace different from the backup component namespace, uninstall this version
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_install.adoc]:
```suggestion
If you manually install the OADP operator, the custom resource definition version of the OADP operator and Velereo must exactly match. If these versions do not exactly match one another, you encounter problems. If you have previously installed and used the OADP Operator on the hub cluster in a namespace different from the backup component namespace, uninstall this version.
```

**Review by dockerymick — COMMENTED** [review]
Left a suggestion

**Review by dockerymick — APPROVED** [review]

**Pull request merged**